include/AEPAssurance.h

```diff
@@ -0,0 +1,40 @@
+//
+//  AEPAssurance.h
+//  Adobe Experience Platform -- iOS Framework
+//
+//  Copyright 1996-2020. Adobe. All Rights Reserved
+//
+//  AEPAssurance Version: 1.0.0
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface AEPAssurance : NSObject
+
+    /**
+     * @brief Registers the AEPAssurance extension with the Core Event Hub.
+    */
++ (bool) registerExtension;
+
+/**
+ * @brief Provides the current version of Assurace SDK.
+ *
+ * @return a string representing the version of Assurace SDK
+*/
++ (nonnull NSString*) extensionVersion;
+
+/**
+ * @brief Starts a Assurace session.
+ *
+ * Calling this method when a session has already been started will result in a no-op.
+ * It will attempt to initiate a new Assurace session if no session is active.
```

Comment:
It's recommended not to use the future tense in docs, you can change this to something like 
```
Calling this method when a session has already been started results in a no-op, otherwise it attempt to initiate a new Assurance session.
```